compiler-core/src/reference.rs

```diff
@@ -403,14 +418,22 @@ impl ReferenceTracker {
         _ = self.graph.add_edge(self.current_node, target, ());
     }
 
-    pub fn register_module_reference(&mut self, name: EcoString) {
+    pub fn register_module_reference_for_unqualified_imports_and_aliases(
```

Comment:
I don't understand this new name. It seems to say something about the context in which it is used, but not name the function itself, and the meaning is unclear.

Could these two functions be given clearer names please, and documentation comments could be useful 🙏

Reply:
You're right, sorry about that. I've given them both better names now, relating to the sort of module name they take as arguments.